src/framework/audio/engine/internal/audioengine.cpp

```
@@ -36,7 +38,7 @@ static constexpr int MAX_SUPPORTED_AUDIO_CHANNELS = 2;
 
 AudioEngine::AudioEngine()
 {
-    m_mixer = std::make_shared<Mixer>();
+    m_context = std::make_shared<AudioContext>(0);
```

Cppcheck (2.20.0)

[performance] Variable 'm_context' is assigned in constructor body. Consider performing initialization in initialization list.

(useInitializationList)